test/expected-failures.txt

```diff
@@ -6,22 +6,13 @@
 # canonicalizing the timezone name.
 staging/Intl402/Temporal/old/date-time-format.js
 
-# temporal-test262-runner doesn't support the $262 global object
-intl402/DateTimeFormat/proto-from-ctor-realm.js
```

Comment:
Note, I added this in https://github.com/js-temporal/temporal-polyfill/pull/282 in order to be able to support https://github.com/js-temporal/temporal-test262-runner/pull/19, but then we decided to stick to npm releases of temporal-test262-runner only. When we update temporal-test262-runner, we'll need to add this back in.